frontend/src/components/Election/Admin/ViewBallots.tsx

```diff
@@ -31,6 +34,26 @@ const ViewBallots = ({ election, permissions }) => {
         return event.toLocaleString();
     }
 
+    const buildCsvData = () => {
+        let header = [
+            { label: 'ballot_id', key: 'ballot_id' },
+            ...data.election.races[0].candidates.map((c) => ({ label: c.candidate_name, key: c.candidate_id }))
```

Comment:
I'm curious, when we extend this to multiple races, which approach do you think would work better?

All races in 1 CSV: we add more columns to the csv to account for all ballots variations
Separate CSVs per race: When user downloads data there's a page to select which race they want to load (or maybe it's all in a single zip)

I'm assuming we'll go with the "Separate CSVs per race" approach? I'm not sure what if there's a precedent for existing elections

Reply:
I've been thinking about that as well. I think the best would be one file per race. That would allow you to add race configuration details to a header that a parser can read as well. I don't know of any precedent either. 
We could also have a link to download the raw json data and not worry about formatting.